woocommerce-free-shipping-progress-bar-block.php

```diff
@@ -44,20 +44,37 @@ function enqueue_frontend_script() {
 function add_attributes_to_block( $attributes = [], $content = '' ) {
 	$escaped_data_attributes = [];
 
+	// Add the custom background color to the attributes array.
+	if ( ! $attributes['backgroundColor'] && $attributes['style']['color']['background'] ) {
```

Comment:
Thanks for pointing this out. I totally missed that, and I'll add some tools, e.g. phpcs, Psalm, etc. to the plugin so that I can spot mistakes like these easier.